startup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+memcached -u memcache -d start
```

Comment:
Hrm ok, cool! But not what we want to do. 

In general, every docker container should contain one process. This is important because we want visibility into when the process fails (the container should then exit with a non-zero code). This way, we can monitor all our containers without needing to know what's going on inside them. We just know it's one process. 

So memcached will be separate!

This is also cool because we'll run it in Kubernetes. Probably via a Helm chart! https://bitnami.com/stack/memcached/helm